xyz-hub-test/src/test/java/com/here/xyz/hub/rest/jobs/JobApiExportVersionIT.java

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (C) 2017-2023 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+package com.here.xyz.hub.rest.jobs;
+
+import static com.here.xyz.httpconnector.util.jobs.Export.ExportTarget.Type.DOWNLOAD;
+import static com.here.xyz.httpconnector.util.jobs.Job.Status.failed;
+import static com.here.xyz.httpconnector.util.jobs.Job.Status.finalized;
+
+import com.here.xyz.httpconnector.util.jobs.Export;
+import com.here.xyz.httpconnector.util.jobs.Job;
+import com.here.xyz.models.geojson.coordinates.PointCoordinates;
+import com.here.xyz.models.geojson.implementation.Point;
+import com.here.xyz.models.geojson.implementation.Properties;
+
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class JobApiExportVersionIT extends JobApiIT {
+
+    protected String testExportJobId = "export-version-job";
+    protected static String scope = "export";
+    protected static String testVersionedSpaceId1 = "version-space-1";
+    protected static String testVersionedSpaceId1Ext = testVersionedSpaceId1 + "-ext";
+    private static int versionsToKeep = 10;
+
+    @BeforeClass
+    public static void init(){
+
+        /** Create test space with CompostitSpace, Version and content */
+        createSpaceWithCustomStorage(getScopedSpaceId(testVersionedSpaceId1, scope), "psql", null, versionsToKeep);
+        postFeature(getScopedSpaceId(testVersionedSpaceId1,scope), newFeature()
+                        .withId("id000")
+                        .withGeometry(new Point().withCoordinates(new PointCoordinates( 8.61,50.020093)))  // base  - htile 122001322003 , 23600771
+                        .withProperties(new Properties().with("group", "shouldBeEmpty")
+                                                        .with("vflag", "oldest")
+                                       ),
+                AuthProfile.ACCESS_OWNER_1_ADMIN
+        );
+        postFeature(getScopedSpaceId(testVersionedSpaceId1,scope), newFeature()
```

Comment:
Looks like quite a lot of duplicated code. Better do this in a loop with specified inputs.